packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -4804,16 +4793,18 @@ function getResourceKey(as: string, href: string): string {
 }
 
 export function prefetchDNS(href: string, options?: mixed) {
```

Comment:
There seems be a lot of these exports that are not actually used anywhere. We should be careful not to export something that is never imported because that won't be caught by lint rules that tell us if they're unused. Especially if they're part of Configs since those have indirections that make them not clear if they're used.

It makes it harder to review too.

These also seem like they can be easily hoisted out into a separate file since they're not actually part of being called from the config. It's whole separate dispatcher that then calls into the runtime.

Reply:
extracted to a dispatcher function